forge-gui/res/cardsfolder/s/sentinel_of_the_pearl_trident.txt

```diff
@@ -3,7 +3,7 @@ ManaCost:4 U
 Types:Creature Merfolk Soldier
 PT:3/3
 K:Flash
-T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigExile | Optional$ True | TriggerDescription$ When CARDNAME enters the battlefield, you may exile target historic permanent you control. If you do, return that card to the battlefield under its owner's control at the beginning of the next end step. (Artifacts, legendaries, and Sagas are historic.)
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigExile | OptionalDecider$ You | TriggerDescription$ When CARDNAME enters the battlefield, you may exile target historic permanent you control. If you do, return that card to the battlefield under its owner's control at the beginning of the next end step. (Artifacts, legendaries, and Sagas are historic.)
 SVar:TrigExile:DB$ ChangeZone | ValidTgts$ Permanent.Historic+YouCtrl | TgtPrompt$ Select target historic permanent you control | Origin$ Battlefield | Destination$ Exile | RememberChanged$ True | SubAbility$ DelTrigEffect
 SVar:DelTrigEffect:DB$ DelayedTrigger | Mode$ Phase | Phase$ End Of Turn | Execute$ TrigReturn | RememberObjects$ RememberedLKI | TriggerDescription$ Return the exiled card to the battlefield under its owner's control at the beginning of the next end step. | SubAbility$ DBCleanup | AILogic$ Always
```

Comment:
should be written as Cost for "If you do" ?

Reply:
can't be done currently because it targets

Reply:
if it targets, then this is okay